README.md

```diff
@@ -1,4 +1,4 @@
-# express-redirect [![Build Status](https://img.shields.io/travis/phvcky/express-redirect.svg?branch=master)](https://travis-ci.org/phvcky/express-redirect) [![NPM Version](https://img.shields.io/npm/v/express-redirect.svg)](https://www.npmjs.org/package/express-redirect)
+# express-redirect [![Build Status](https://img.shields.io/travis/buschtoens/express-redirect.svg?branch=master)](https://travis-ci.org/buschtoens/express-redirect) [![NPM Version](https://img.shields.io/npm/v/express-redirect.svg)](https://www.npmjs.org/package/express-redirect) [![gandalf](http://img.shields.io/badge/gandalf-approved-61C6FF.svg)](https://www.youtube.com/watch?v=Sagg08DrO5U)
```

Comment:
Thanks for fixing that. However that should be moved its own housekeeping PR, that may also contain the `.gitignore` additions and the contributors amendment.